### ISSUE TITLE
Configuration values of "information" produce warnings instead of informational messages in the command line

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#1143]
**Describe the bug**
Configuration values of "information" produce "warning" type messages in the command line.

**To Reproduce**
Set up a Pyright project with a `foo.py` containing:

```py
# pyright: reportCallInDefaultInitializer=information

def foo(value: None = breakpoint()):
    ...
```

Then run:

```
npx pyright@1.1.83 foo.py
```

... and observe that Pyright produces a warning:

```
[...]
  3:23 - warning: Function calls within default value initializer are not permitted (reportCallInDefaultInitializer)
0 errors, 0 warnings, 1 info
```

**Expected behavior**
That Pyright would produce "information" messages, like in VS Code, for "information" type settings.

**VS Code extension or command-line**
CLI, 1.1.83

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.84, which I just published.
